sass/components/rtl_support/_badges.scss

```diff
@@ -0,0 +1,17 @@
+// RTL Badges
+.rtl {
+	span.badge {
+		margin-left: initial;
+		margin-right: 14px;
+		float: left;
+
+		&.new:after {
+			content: " جدید";
```

Comment:
@arminmon I guess you are right. I will create a PR that fixes that. The idea with the attribute sounds great but there is currently no or limited support for that feature :( http://caniuse.com/#feat=css3-attr